packages/utils/src/metrics.js

```diff
@@ -0,0 +1,129 @@
+// @ts-check
```

Comment:
maybe make single file for getAddress and getFinger? and not store all in one place
like it done for keystore files